.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Add `.vscode/` to your `.gitignore` so you're not forcing editor changes on other devs 🙂